Sources/SwiftSyntax/generated/raw/RawSyntaxValidation.swift

```diff
@@ -2644,11 +2613,8 @@ func validateLayout(layout: RawSyntaxBuffer, as kind: SyntaxKind) {
     assertNoError(kind, 1, verify(layout[1], as: RawTokenSyntax.self))
     assertNoError(kind, 2, verify(layout[2], as: RawUnexpectedNodesSyntax?.self))
     assertAnyHasNoError(kind, 3, [
-      verify(layout[3], as: RawSyntax.self),
-      verify(layout[3], as: RawSyntax.self),
-    ])
+        verify(layout[3], as: RawSyntax.self)])
     assertNoError(kind, 4, verify(layout[4], as: RawUnexpectedNodesSyntax?.self))
     break
-  }
-#endif
+  }#endif 
```

Comment:
I’m surprised this is even syntactically valid. Do you think you could try to put `#endif` on its own line in a follow-up PR?